hack/dockerized

```diff
@@ -107,9 +107,15 @@ _rsync \
 
 volumes="-v ${BUILDER}:/root:rw,z"
 
-# append .docker directory as volume
-mkdir -p "${HOME}/.docker"
-volumes="$volumes -v ${HOME}/.docker:/root/.docker:ro,z"
+# Copy latest docker/podman auth file and append .docker directory as volume
+mkdir -p ".docker"
+ID=$(id -u)
+if [[ $KUBEVIRT_CRI = "podman" ]] && [[ -f "/tmp/podman-run-${ID}/containers/auth.json" ]]; then
+    cp "/tmp/podman-run-${ID}/containers/auth.json" .docker/config.json
+elif [[ -f "${HOME}/.docker/config.json" ]]; then
+    cp "${HOME}/.docker/config.json" .docker/config.json
+fi
+volumes="$volumes -v $(pwd)/.docker:/root/.docker:ro,z"
```

Comment:
i feel a bit weird about copying auth files around automatically into different directories.

What woudl be wrong with changing the host path on the volume to the podman authfile when podman is in use?

Reply:
Yeah I didn't like writing it either!
podman names the file differently (auth.json vs config.json), so I don't know how else to do this...

Reply:
Maybe we can do a single file volume or a bind mount to work around that, I'll give it a shot tomorrow.

Reply:
Force-pushed a bind mount, seems to do the trick!